packages/zoe/test/swingsetTests/zoe/vat-bob.js

```diff
@@ -133,7 +133,7 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
         bobPayments,
       );
 
-      const bobOutcome = await E(seat).exercise();
+      const bobOutcome = await E(seat).makeOffer();
```

Comment:
Hmm, I don't like changing all of these to `makeOffer`. That seems to defeat the purpose of having a seat.